sys/include/net/ng_pkt.h

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (C) 2014 Martine Lenders <mlenders@inf.fu-berlin.de>
```

Comment:
I was once told that I should always update my copyrights to whenever I touched the file last - but I don't really care...